api/desecapi/tests/test_rrsets.py

```diff
@@ -1625,6 +1625,8 @@ def assertRequests(*, allowed):
 class AuthenticatedRRSetLPSTestCase(AuthenticatedRRSetBaseTestCase):
     DYN = True
```

Comment:
do we have tests for `DYN = False`?

Reply:
yes, that's the default, and the tests are in class `AuthenticatedRRSetTestCase`